From jirislaby@gmail.com  Fri Jul 10 15:15:05 2009
From: Jiri Slaby <jirislaby@gmail.com>
Date: Fri,  3 Jul 2009 23:09:41 +0200
Subject: USB: usb-serial, remove unused variables
To: gregkh@suse.de
Message-ID: <1246655381-18007-1-git-send-email-jirislaby@gmail.com>


There are some unused variables in serial_do_down. Remove them.

Signed-off-by: Jiri Slaby <jirislaby@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/usb-serial.c |    6 ------
 1 file changed, 6 deletions(-)

--- a/drivers/usb/serial/usb-serial.c
+++ b/drivers/usb/serial/usb-serial.c
@@ -284,8 +284,6 @@ bailout_serial_put:
 static void serial_do_down(struct usb_serial_port *port)
 {
 	struct usb_serial_driver *drv = port->serial->type;
-	struct usb_serial *serial;
-	struct module *owner;
 
 	/* The console is magical, do not hang up the console hardware
 	   or there will be tears */
@@ -293,12 +291,8 @@ static void serial_do_down(struct usb_se
 		return;
 
 	mutex_lock(&port->mutex);
-	serial = port->serial;
-	owner = serial->type->driver.owner;
-
 	if (drv->close)
 		drv->close(port);
-
 	mutex_unlock(&port->mutex);
 }
 
